web-animations-1/Overview.bs

```diff
@@ -847,14 +847,14 @@ becomes <a lt="idle play state">idle</a>, is when the procedure to
 
 </div>
 
-<h4 id="setting-the-target-effect">Setting the target effect of an
+<h4 id="setting-the-associated-effect">Setting the associated effect of an
```

Comment:
Unfortunately there isn't, I'm changing the id back for now.

Reply:
@stephenmcgruer What do you think?

Reply:
The [bikeshed docs](https://tabatkins.github.io/bikeshed/#id-gen) say:
> If a heading changed significantly, so that you want to change the ID, but you want links to the old heading ID to still work, put the old ID in an oldids="" attribute on the heading element. If there are multiple, comma-separate them.